Failure was:
```
00:43:00.234 kafka.server.DynamicBrokerReconfigurationTest > testAddRemoveSslListener FAILED
00:43:00.234     java.lang.AssertionError: expected:<10> but was:<11>
00:43:00.234         at org.junit.Assert.fail(Assert.java:88)
00:43:00.234         at org.junit.Assert.failNotEquals(Assert.java:834)
00:43:00.234         at org.junit.Assert.assertEquals(Assert.java:645)
00:43:00.234         at org.junit.Assert.assertEquals(Assert.java:631)
00:43:00.234         at kafka.server.DynamicBrokerReconfigurationTest.verifyProduceConsume(DynamicBrokerReconfigurationTest.scala:956)
00:43:00.234         at kafka.server.DynamicBrokerReconfigurationTest.verifyListener(DynamicBrokerReconfigurationTest.scala:827)
00:43:00.234         at kafka.server.DynamicBrokerReconfigurationTest.testAddRemoveSslListener(DynamicBrokerReconfigurationTest.scala:703)
```
(https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/12005)

retest this please

--- a/build.gradle
+++ b/build.gradle
@@ -923,6 +923,7 @@ project(':streams') {
     testCompile project(':clients').sourceSets.test.output
     testCompile project(':core')
     testCompile project(':core').sourceSets.test.output
+    testCompile project(':streams:test-utils').sourceSets.main.output
     testCompile libs.junit
     testCompile libs.easymock
     testCompile libs.bcpkix
@@ -967,11 +968,12 @@ project(':streams:test-utils') {
   archivesBaseName = "kafka-streams-test-utils"
 
   dependencies {
-    compile project(':streams')
+    compile project(':streams').sourceSets.main.output
     compile project(':clients')
 
     testCompile project(':clients').sourceSets.test.output
     testCompile libs.junit
+    testCompile libs.rocksDBJni
 
     testRuntime libs.slf4jlog4j
   }

